chia/plotters/bladebit.py

```diff
@@ -211,16 +211,13 @@ def plot_bladebit(args, chia_root_path, root_path):
         return
 
     version = None
+    actual_version = None
```

Comment:
While I can follow the logic here to see how disk or RAM plotting is determined, it does feel a bit odd.

Just to explore an alternative, if we had an enum that offered `DISK` or `RAM` plotting options:

```
class BBPlottingType(Enum):
  RAM = 'ramplot' # Supported by BB1 and BB2
  DISK = 'diskplot' # BB2 only
```

Then I think you could do something like:

```
plotting_type = BBPlottingType.RAM # Assume RAM plotting by default

if args.plotter == "bladebit2":
  plotting_type = BBPlottingType.DISK

...

call_args.append(plotting_type.value)
```

Would that work?

Reply:
Actually, command syntax of bladebit1 and bladebit2 with ramplot option are a bit different.

### Bladebit 1 (Always RAM plot)
```
bladebit <general options>
```

### Bladebit 2 with RAM plot option
```
bladebit <general options> ramplot [additional options]
```

If you want to do ramplot with bladebit2, you need to specify `ramplot` flag while bladebit1 doesn't recognize it.

And to make the code simple, when you want to do ramplot with bladebit2, GUI posts request to chia daemon as if there is bladebit1 installed, because whole option syntax is the same with bladebit1 except for `ramplot` flag.

Maybe it makes easy to understand how GUI recognize ramplot option when bladebit2 is installed.
![image](https://user-images.githubusercontent.com/84098616/200351089-5d9765fd-eb57-423f-babf-29a5e14e7a2d.png)